lib/tasks/bulkupload.rake

```diff
@@ -0,0 +1,89 @@
+namespace :bulkupload do
+  desc "TODO"
```

Comment:
nit: would be good if we could have a basic description for these tasks, ideally mentioning `BULK_UPLOAD_PATHS` with an example value (something like `BULK_UPLOAD_PATHS="<path>,<path>" bundle exec rake bulkupload:s3` sort of thing).

You can do multi-line descriptions:

```
  desc <<~DESC
    Imports all relevant entities from the ProjectWorks API, excluding timesheets by default.

    If you wish to import timesheets, set an environment variable named
    "PW_INCLUDE_TIMESHEETS_IN_IMPORT" with a value of "yes".
  DESC
```

`rake` will render these nicely by only showing the first line when outputting in a table format, and then show the full description if you ask for the long format list.